creator/Documents/CommandBlocks.md

```diff
@@ -25,127 +25,116 @@ In this guide, we will set up a command block chain to look for a certain type o
 
-The Scoreboard is a gameplay system that exists within Minecraft that can keep track of players, objectives, and score values. The scoreboard enables a wide use of options to create unique gameplay styles. In this tutorial, you will have a single objective and value in order to track when a player has completed the task.
-
-1. In Minecraft, open up the chat dialogue box (T or Enter on Windows 10 OS)
-1. Enter the following command: `/scoreboard objectives add wool_placed dummy`.
-    1. The command is adding a brand new objective called Wool_Placed. Dummy fills the argument since scores can only be adjusted via commands.
-1. Enter the following command: `/scoreboard players set @p wool_placed 0`.
+1. Check to see if a diamond block exists at a set location.
+1. If it does, check to see if the player has already received the reward.
+1. If the player hasn't received the reward, give the player an emerald.
+1. After giving the player an emerald, mark that they have received the reward.
 
 ### Setting up the first Command Block
 
 Now that we have looked at the settings of a command block, let’s set up the first block to detect a certain type of block in a location.
 
 1. Place a command block on the ground in a location near your player character.
-1. Right-click to open up the command block GUI.
+1. Open up the command block interface.
 1. Set the **Block Type** to **Repeat**.
-1. Set the **Condition** to **Unconditional**.
 1. Set the **Redstone** to **Always Active**.
 
-#### The /testforblock Command
+#### The `/testforblock` Command
 
-The /testforblock command will look for a specific block at a specific point that the creator can specify. In the example of `/testforblock 0 0 0 wool`, you can see that the command is looking for any type of wool block located at 0, 0, 0.
+The `/testforblock` command looks for a specific block at a specified point in the world.
 
-`/testforblock <position: x y z> <tilename:block> [datavalue: int]`
+`/testforblock <position: x y z> <tileName: Block> [dataValue: int]`
 
-- **position** is the coordinates in the world where the command block will look for the specific block.
-- **tilename** is the type of block you are looking for. This must be the Bedrock Block ID of the block.
-- **datavalue** is an extra argument used to handle variations of the same block. For instance, wool has many different colors. If you set it to `wool` or `wool -1`, it will look for all types of wool. If you set it to `wool 1`, it will only look for orange wool.
+- **position** is the coordinates in the world where the command will look for the specific block. Example: `0 4 0`.
+- **tileName** is the type of block you are looking for. Example: `diamond_block`.
+- **dataValue** is an optional argument used to handle variations of the same block. For this tutorial, we won't be needing to use this.
```

Comment:
I changed the capitalization to match how bedrock's in-game command suggestions list those arguments. I also modified the stuff about data values because data values are really outdated legacy stuff that is really not nice for beginners to learn, glossing over the fact that 1 = orange in the original just raises so many unanswered questions that are totally out of scope of a simple beginner tutorial.